docs/getting-started.md

```diff
@@ -41,6 +41,8 @@ npm install next react react-dom
 yarn add next react react-dom
 ```
 
+> **Note**: [Yarn 2](https://yarnpkg.com/getting-started/install) is supported as best-effort within Next.js. For best results, use NPM or Yarn 1.
```

Comment:
Yarn 2 !== Yarn PnP, just like with Yarn 1 you can use both PnP and node_modules so this should really be updated as this isn't accurate. The [migration guide](https://yarnpkg.com/getting-started/migration) explicitly states to start with `node_modules` then enable PnP once ready

Reply:
The issue that sprung this PR was multiple place on GitHub it's "Yarn 2 is supported". By _default_ Yarn 2 will enable PnP. So yes, I agree 100% with your statement, but that's not what in practice will happen when people try to use Yarn 2.

Reply:
> but that's not what in practice will happen when people try to use Yarn 2.

If people skip the migration guide I'd argue that is their problem and not something that should cause Next.js to advise against Yarn 2

Reply:
In essence I agree with you, but that's not reality. Yarn 2 _defaults_ to PnP. There is no call to action to migrate. Next.js just states "it works with Yarn 2", not "You must follow the migration guide". 

It's very easy to say that it's other people's problem, but it's equally elitist to do so.